docs/faq/code-analysis/which-metrics-does-codacy-calculate.md

```diff
@@ -0,0 +1,114 @@
+---
+description: Codacy scans your code for issues and calculates the metrics code complexity, duplication, and coverage. Besides this, Codacy also calculates a grade for your repository and files based on all calculated code quality metrics.
+---
+
+# Which metrics does Codacy calculate?
+
+Codacy performs static code analysis and calculates code duplication, code complexity, and code coverage metrics for [most supported programming languages](../../getting-started/supported-languages-and-tools.md).
+
+The following sections describe how Codacy calculates each supported metric and where you can see each metric on the Codacy UI:
+
+-   [Grade](#grade)
+-   [Issues](#issues)
+-   [Complexity](#complexity)
+-   [Duplication](#duplication)
+-   [Code coverage](#code-coverage)
+
+## Grade
+
+Codacy assigns an overall grade to your repository branches and to individual files to help you understand the code quality of your repository. Grades depend on a score that takes into account a weighted average of all supported code quality metrics, and range from **A** (highest grade) to **F** (lowest grade):
+
+|Grade                          |Score     |
+|-------------------------------|----------|
+|![Grade A](images/grade_a.png) |91 to 100 |
+|![Grade B](images/grade_b.png) |71 to 90  |
+|![Grade C](images/grade_c.png) |51 to 70  |
+|![Grade D](images/grade_d.png) |31 to 50  |
+|![Grade E](images/grade_e.png) |11 to 30  |
+|![Grade F](images/grade_f.png) |0 to 10   |
+
+Codacy displays grades on the following pages:
+
+-   [**Files page**](../../repositories/files.md): displays the grade for each file in your repository
+-   [**Branches page**](../../repositories-configure/managing-branches.md): displays the grade for each analyzed branch in your repository
+-   [**Repository Dashboard**](../../repositories/repository-dashboard.md): displays the overall grade of your repository
+-   [**Organization Overview**](../../organizations/organization-overview.md): displays which repositories have each grade
+-   [**Repositories list page**](../../organizations/managing-repositories.md): displays the grade of each repository in your organization
+
+## Issues
+
+Codacy calculates both the number of issues and the percentage of issues in your repository when compared to an industry average, taking into account the severity of each issue:
```

Comment:
This part abstracts some details that aren't yet clear enough, see [this comment on CY-2086](https://codacy.atlassian.net/browse/CY-2086?focusedCommentId=42199).

Reply:
@prcr Is this referring to the issues severity showed in Codacy UI as well? - severity minor, medium and critical. 
If yes, let's keep the naming consistent.

Reply:
You're right, we renamed the severity levels at one point but I was working with information older than that. It's fixed now! :raised_hands: